rindexer_rust_playground/src/rindexer_lib/typings/rindexer_playground/events/rocket_pool_eth.rs

```diff
@@ -145,9 +145,9 @@ where
         Fut: Future<Output = EventCallbackResult<()>> + Send + 'static,
     {
         let csv = AsyncCsvAppender::new(
-            r"/Users/jackedgson/Development/avara/rindexer/rindexer_rust_playground/generated_csv/RocketPoolETH/rocketpooleth-approval.csv",
+            r"/Users/skanda/Projects/rindexer/cli/../rindexer_rust_playground/generated_csv/RocketPoolETH/rocketpooleth-approval.csv",
```

Comment:
@joshstevens19 not sure about this one but I ran the makefile for codegen typings because my changes needed to regenerate it (I added state_notifications into NetworkContract)

both prev and current strings are dev directory lists so maybe this can go in as it is?